docs/cli-arguments.md

```diff
@@ -26,6 +26,7 @@
 | `--enable-controller-ingress-class-networkingv1` | `bool` | Enable the networking.k8s.io/v1 IngressClass controller. | `true` |
 | `--enable-controller-ingress-class-parameters` | `bool` | Enable the IngressClassParameters controller. | `true` |
 | `--enable-controller-ingress-networkingv1` | `bool` | Enable the networking.k8s.io/v1 Ingress controller. | `true` |
+| `--enable-controller-kong-custom-entity` | `bool` | Enable the KongCustomEntity controller. | `true` |
```

Comment:
I see this being set as enabled by default without a feature flag.

Just wanted to make sure that's this is the direction we wanted to go. Enabling a just introduced alpha API by default seems rushing it a bit too me. The linked issue https://github.com/Kong/kubernetes-ingress-controller/issues/5976 doesn't seem to mention this decision.

WDYT about making this disabled by default?

Reply:
Set this disabled by default and added a comment for further discussion.